src/components/PathwayDisplay/PathwayDisplay.jsx

```diff
@@ -0,0 +1,23 @@
+import "./PathwayDisplay.scss";
+import hamburger from "../../assets/hamburger-icon.svg";
+import grid from "../../assets/grid-icon.svg";
+import Button from "../Button/Button";
+import {ReactComponent as Pencil} from "../../assets/pen-icon.svg";
+import { Link } from "react-router-dom";
+
+
+const PathwayDisplay = ({toggleDisplay}) => {
+  return (
+    <div className="pathway-display">
+        <div className="pathway-display__container">
+          <img className="pathway-display__option" id="grid-icon" src={grid} alt="grid display option button" onClick={() => toggleDisplay(true)}/>
+          <img className="pathway-display__option" src ={hamburger} alt="list view option button" onClick={() => toggleDisplay(false)}/>
```

Comment:
Can we change so that selected changes the border and opacity what is selected, as discussed :D